src/main/java/org/isf/agetype/manager/AgeTypeBrowserManager.java

```diff
@@ -113,10 +113,12 @@ public AgeType getTypeByCode(String code) throws OHServiceException {
 	protected void validateAgeTypes(List<AgeType> ageTypes) throws OHDataValidationException {
 		List<OHExceptionMessage> errors = new ArrayList<>();
 		for (int i = 1; i < ageTypes.size(); i++) {
-			if (ageTypes.get(i).getFrom() <= ageTypes.get(i - 1).getTo()) {
+			boolean isOverlapping = ageTypes.get(i).getFrom() <= ageTypes.get(i - 1).getTo();
+			boolean isGapLargerThanOne = ageTypes.get(i).getFrom() - ageTypes.get(i - 1).getTo() > 1;
+			if (isOverlapping) {
```

Comment:
One of the first things I was taught during my many decades of writing commercial software is this "rule":

1. Write something once and leave it as is
2. Write something twice and if it is simple leave it as is; if it is complex then consider exacting it into a method and be willing to incur the overhead
3. Write something three times, then exact it into a method as the other head and garbage collection is worth the cost so there is consistency

Thus I find most/all the occurrences of extracting into local variables and methods unnecessary. 

I don't approve of the changes.